README.md

```diff
@@ -1 +1,205 @@
-# to-do-list
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 To Do list <a name="about-project"></a>
+
+**Awesome-books** this is a website that help family and people to organize to do list of the day
+
+## 🛠 Built With <a name="built-with"> </a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Html</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Css</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Javascript</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[to do list]**
+- **[add button]**
+- **[remove button]**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LIVE DEMO -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+A working connection and a coding editor like visual studio code
+
+### Setup
+
+Clone this repository to your desired folder:
+Cd folder
+Git clone (https://github.com/kerlos-bahaa/To-Do-list-list-structure)
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+Use git push at your desired location and use the command git clone and paste the URL
+
+### Install
+
+
+Install this project with:
+
+- You need to install Chrome if you want to see it working locally
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+- npm start
+
+### Run tests
+
+To run tests, run the following command:
+
+- npx stylelint "\*_/_.{css,scss}"
+- npx hint .
+- npx eslint .
+
+### Deployment
+
+You can deploy this project using:
+
```

Comment:
- [ ] Since your work is not deployed, kindly delete this section, or give a detailed step on how to deploy your project just as suggested by the last reviewer 👍